tests/test_waterwheel.py

```diff
@@ -32,7 +32,24 @@ def test_proper_nouns(self):
         self.assertEqual(str(doc.ents[1].label_), 'RIVER')
         for ent in doc.ents:
             self.assertIsNotNone(ent._.wikilink)
-        
+
+        doc = self.nlp('This image of the Pacific Ocean seafloor and land elevations was created by the World Data Center.')
+        self.assertEqual(str(doc.ents[0]), 'Pacific Ocean')
+        self.assertEqual(str(doc.ents[0].label_), 'OCEAN')
+        doc = self.nlp('This image of the Pacific Ocean seafloor and land elevations was created by the World Data Center.')
+        self.assertEqual(str(doc.ents[0]), 'pacific sea')
+        self.assertEqual(str(doc.ents[0].label_), 'OCEAN')
+        for ent in doc.ents:
+            self.assertIsNotNone(ent._.wikilink)
+
+        doc = self.nlp('Manitoba is bordered by Ontario.')
+        self.assertEqual(str(doc.ents[0]), 'Manitoba')
+        self.assertEqual(str(doc.ents[0].label_), 'PROVINCE')
+        self.assertEqual(str(doc.ents[1]), 'Ontario')
+        self.assertEqual(str(doc.ents[1].label_), 'PROVINCE')
+        for ent in doc.ents:
+            self.assertIsNotNone(ent._.wikilink)
+
```

Comment:
Add test cases here that check whether entities with proper noun format are matched with and without a qualifier and if entitry which is all lower letters or all capitals is NOT matched without a qualifier.